1558578652552	Sync.SyncScheduler	DEBUG	Next sync in 600000 ms. (why=schedule)
1558578678596	Sync.RemoteTabs	INFO	Generating tab list with filter
1558578678597	Sync.RemoteTabs	INFO	Final tab list has 0 clients with 0 tabs.
1558578721738	Sync.SyncScheduler	DEBUG	Network link status changed to "down". Offline?: false
1558578721739	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1558578721741	Sync.SyncScheduler	DEBUG	Next sync in 600000 ms. (why=schedule)
1558578722570	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1558578722820	Sync.SyncScheduler	DEBUG	Network link status changed to "up". Offline?: false
1558578722820	Sync.SyncScheduler	DEBUG	Network link looks up. Syncing.
1558578722830	Sync.Service	DEBUG	User-Agent: Firefox/67.0 (Linux x86_64) FxSync/1.69.0.20190521202118.desktop
1558578722830	Sync.Service	INFO	Starting sync at 2019-05-22 20:32:02 in browser session feW6KSQ4L3zF
1558578722831	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1558578722836	Sync.Status	INFO	Resetting Status.
1558578722836	Sync.Status	DEBUG	Status.service: success.status_ok => success.status_ok
1558578722849	Sync.Resource	WARN	GET request to https://sync-548-us-west-2.sync.services.mozilla.com/1.5/84144729/info/collections failed: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1558578722849	Sync.Status	DEBUG	Status.sync: success.sync => error.login.reason.network
1558578722849	Sync.Status	DEBUG	Status.service: success.status_ok => error.sync.failed
1558578722850	Sync.ErrorHandler	ERROR	Sync encountered an error: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1558578722851	Sync.SyncScheduler	DEBUG	Next sync in 600000 ms. (why=reschedule)
1558578722856	Sync.Service	DEBUG	Exception calling anonymous function: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
